### PR TITLE
Added ability to embed Vimeo videos

### PR DESCRIPTION
**Pull request by jrkd**
### Code changes
- Renamed YoutubeVideo[xxx] files, classes and comments to Video[xxx]
- Added tests for vimeo urls
- Added a new REGEX_VIDEO_URL that checks either youtube or vimeo url formats.
- Added a condition to VideoHelper.GetVideoId to get youtube ids separately to vimeo.
- Added GetVideoEmbedFormat to VideoHelper and replaced the explicit mention of youtube's embed url in the VideoWidget View.

### Motivation

This is adding the ability for the Video widget to render Vimeo videos rather than just Youtube videos.

### Checklist

- [x] Code follows coding conventions held in this repo
- [x] Automated tests have been added
- [x] Tests are passing
- [ ] Docs have been updated (if applicable)
- [x] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

Add a youtube video to the sandbox project, then add a vimeo video to the project. Both should render.

### CODE REVIEW

File: Kentico.Widget.Video/Helpers/VideoHelper.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+            // return last segment in absolute path as a video identifier
+            return videoUri.AbsolutePath.Split('/').LastOrDefault();
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="videoUrl"></param>
+        /// <returns></returns>
```
[the remark this answers is not in the document]
Comment (jrkd) [reply Kentico.Widget.Video/Helpers/VideoHelper.cs]: Have added docs & unit tests for this method.

File: Kentico.Widget.Video.Views/Views/Shared/Kentico/Widgets/_VideoWidget.cshtml [hunk 2]
```diff
@@ -8,14 +8,16 @@
 @{
     var widgetGuid = Guid.NewGuid();
     var videoUrl = !string.IsNullOrWhiteSpace(Model.VideoUrl)
-        ? $"https://www.youtube.com/embed/{YoutubeVideoHelper.GetVideoId(Model.VideoUrl)}"
+        ? $"{VideoHelper.GetVideoEmbedFormat(Model.VideoUrl)}{VideoHelper.GetVideoId(Model.VideoUrl)}"
```
[the remark this answers is not in the document]
Comment (jrkd) [reply Kentico.Widget.Video.Views/Views/Shared/Kentico/Widgets/_VideoWidget.cshtml]: Have left as is for now.

**Comment by jrkd** [comment]
Hi @OndraM-Kentico, anything else needed here or are you happy to merge?

**Comment by OndraM-Kentico** [comment]
I've just been waiting for approval of the Product owner and in the meantime I've done little refactoring and pushed it into your repo. We can merge it altogether in one PR.

**Pull request merged**